packages/buidler-core/src/internal/buidler-evm/provider/node.ts

```diff
@@ -1044,7 +1069,7 @@ export class BuidlerNode extends EventEmitter {
         header: {
           gasLimit: this._blockGasLimit,
           nonce: "0x42",
-          timestamp: await this._getNextBlockTimestamp()
+          timestamp: await this._getNextUsableBlockTimestamp()
```

Comment:
Now that I see this here. Shouldn't this be enough?

re: https://github.com/nomiclabs/buidler/pull/508/files#r403510853